dev/benchmarks/complex_layout/test/measure_scroll_smoothness.dart

```diff
@@ -68,40 +68,28 @@ class ResampleFlagVariant extends TestVariant<TestScenario> {
 
   late TestScenario currentValue;
   bool get resample {
-    switch (currentValue) {
-      case TestScenario.resampleOn90Hz:
-      case TestScenario.resampleOn59Hz:
-        return true;
-      case TestScenario.resampleOff90Hz:
-      case TestScenario.resampleOff59Hz:
-        return false;
-    }
+    return switch (currentValue) {
```

Comment:
nit while you're at it, you can make this an arrow function:

```dart
  bool get resample => switch (currentValue) {
    TestScenario.resampleOn90Hz || TestScenario.resampleOn59Hz => true,
    TestScenario.resampleOff90Hz || TestScenario.resampleOff59Hz => false,
  };
```

Reply:
Yeah, that looks even better!

But it might be good to keep it as-is to follow the [style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#consider-using--for-short-functions-and-methods).

> only use `=>` when everything, including the function declaration, fits on a single line.

Reply:
the next rule is a further exception for list and map literals, and which I would argue switch expressions should be added: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#use--for-inline-callbacks-that-just-return-list-or-map-literals

Reply:
@christopherfujino The rule seems ambiguous to me

> then rather than using braces and a return` [sic.] statement, _you can_ instead use the ⇒ form

The use of "can" in body of the rule makes the rule sound optional. However, the title makes this seems non-optional. Given this, I won't be retracting my approval on this PR despite me just learning about this rule.